### ISSUE TITLE
Didn't Manage Existing Proton Installations properly

### ISSUE THREAD

**Issue by Jeff-WuYo** [auyer/Protonup-rs#28]
Hi,\
if I didn't select *Detect All*, the program just exit with exit code 0, and didn't prompt me about existing installations. If I select *Detect All*, it works normally.
```
> ProtonUp Menu: Choose your action: Manage Existing Proton Installations
? Select the Applications you want to manage :  
  [ ] Detect All
> [x] Steam "Native" 
  [ ] Steam Flatpak
  [ ] Lutris "Native"
  [ ] Lutris Flatpak
> Select the Applications you want to manage : Steam "Native"
```
A very nice program btw, thx.

**Comment by auyer** [comment]
Oh, I was able to reproduce this bug instantly.
I will look into it soon.
Thanks for pointing it out!

**Comment by auyer** [comment]
Thanks @Jeff-WuYo ! I released a new version [v0.8.3](https://github.com/auyer/Protonup-rs/releases/tag/v0.8.3) with a fix!

**Comment by Jeff-WuYo** [comment]
Yep, it seems behaving properly, build from AUR. Thanks for the fix.
